### PR TITLE
gh-74706: IDLE: Add navigate bar and replace current goto dialog

### PR DESCRIPTION
**Pull request by mlouielu**
* Issue: gh-74706

### CODE REVIEW

File: Lib/idlelib/editor.py [hunk 1]
```diff
@@ -565,16 +568,10 @@ def replace_event(self, event):
...
-        lineno = tkSimpleDialog.askinteger("Goto",
-                "Go to line number:",parent=text)
-        if lineno is None:
-            return "break"
-        if lineno <= 0:
-            text.bell()
-            return "break"
-        text.mark_set("insert", "%d.0" % lineno)
-        text.see("insert")
+        from idlelib.singlebar import NavigateBar
```
Comment (terryjreedy) [inline Lib/idlelib/editor.py]: You named the file navigatebar

**Review by terryjreedy — CHANGES_REQUESTED** [review]
As I will explain on the issue, I do not like the concept and am inclined to reject this.

+ a bar with no title is confusing.

**Comment by brettcannon** [comment]
To try and help move older pull requests forward, we are going through and backfilling 'awaiting' labels on pull requests that are lacking the label. Based on the current reviews, the best we can tell in an automated fashion is that a core developer requested changes to be made to this pull request.

If/when the requested changes have been made, please leave a comment that says, `I have made the requested changes; please review again`. That will trigger a bot to flag this pull request as ready for a follow-up review.

**Comment by erlend-aasland** [comment]
> As I will explain on the issue, I do not like the concept and am inclined to reject this.

@terryjreedy, is this still your stance? If so, I'd like to mark this as pending close.
@mlouielu, are you still interested in pushing these changes?

**Comment by terryjreedy** [comment]
I will try my alternate plan first (see issue), and even that is low priority.  (I believe I have not heard from Louie since Sept. 2017.)

**Pull request closed without merging**